code/game/objects/items/his_grace.dm

```diff
@@ -93,6 +93,7 @@
 				master.remove_status_effect(STATUS_EFFECT_HISGRACE)
 				REMOVE_TRAIT(src, TRAIT_NODROP, HIS_GRACE_TRAIT)
 				master.Knockdown(60)
+				master.adjustStaminaLoss(-5)
```

Comment:
This should be in /datum/status_effect/his_grace/tick() along with the other healing effects. If it's here it'll apply the stamina loss upon the user being eaten by His Grace.

Reply:
Updated per discord discussion